src/pages/workspaces/workspace/DashboardPublic.js

```diff
@@ -57,7 +57,7 @@ const DashboardPublic = ({ namespace, name }) => {
       ]),
       div({ style: Style.dashboard.rightBox }, [
         div({ style: signInStyle }, [
-          div(['Sign in to view full workspace']),
+          div(['Log in to view full workspace']),
```

Comment:
Why the change in terminology? I think we use "Sign in" everywhere, and I'd expect to have to change this is many more places to keep it all consistent.

Reply:
Yeah `Log in` came from Lou. (Trying to find the wireframes.. I can't access invision anymore for some reason.) I updated the button on the dashboard and in the hamburger menu.